data_structures/bloom_filters/bloom_filter/bloom_filter.h

```diff
@@ -0,0 +1,76 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#pragma once
```

Comment:
IIRC #pragma once is non standard, but since practically every compiler supports it now it's probably not a problem. Also [the contribution guideline](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md) requires that you document what these headers are for